packages/html/src/splitter/splitter-pane.tsx

```diff
@@ -0,0 +1,59 @@
+import { classNames } from '../misc';
+
+const SPLITTERPANE_CLASSNAME = 'k-pane';
+
+const states = [];
+
+const options = {};
+
+export type KendoSplitterPaneProps = {
+    collapsible?: boolean;
+    scrollable?: boolean;
+    resizable?: boolean;
+    flexBasis?: string;
+    flex?: boolean;
+};
+
+export type KendoSplitterPaneState = { [K in (typeof states)[number]]?: boolean };
+
+const defaultProps = {
```

Comment:
I checked all Kendo UI suites and there by default Splitter panes are scrollable. In my opinion we should set _scrollable: true;_ as default prop (as it is related to k-scrollable class).